lib/streamlit/config.py

```diff
@@ -49,7 +50,7 @@
 _config_options_template: Dict[str, ConfigOption] = OrderedDict()
 
 # Stores the current state of config options.
-_config_options: Optional[Dict[str, ConfigOption]] = None
+_config_options: Dict[str, ConfigOption] | None = None
```

Comment:
nit: I think with `__future__.annotations` this can be `dict[str, ConfigOption]`

Reply:
I updated all uses of `Dict` and `List` in the changed files 👍  We probably need some kind of regex script to do that across the full code-base in one go.